### ISSUE TITLE
Nx workspace cannot run WallabyJS

### ISSUE THREAD

**Issue by jkyoutsey** [wallabyjs/public#3121]
### Issue description or question
When I run Wallaby with automatic config on my project that uses Karma after converting it to Nx I get this error (including the error where I cannot crate a diagnostics report!)
​[Error] Failed to load configuration file: Automatic Jest configuration error: Module jest-cli is not found in '.'.​​
​​[Error] We've also tried to automatically configure Wallaby.js for other frameworks​​
​​[Error] Automatic Vitest configuration error: Vitest dependency not found.​​
​​[Error] Please refer to our docs: https://wallabyjs.com/docs/config/overview.html ​​
​[Info]​ Started Wallaby.js Core v1.0.1352
​​[Error] Failed to start copyDiagnosticsReportToClipboard operation, TypeError: Cannot read properties of undefined (reading 'map')​​
​​[Error]     at _0x2b15a4 (/Users/jyoutsey/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallabye84dcb/node_modules/ws/lib/Sender.fallbacks.js:1:174347)​​
​​[Error]     at _0x4a82d3.<computed>.<computed> [as copyDiagnosticsReportToClipboard] (/Users/jyoutsey/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallabye84dcb/node_modules/ws/lib/Sender.fallbacks.js:1:174734)​​
​​[Error]     at Object.i.receive (/Users/jyoutsey/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallabye84dcb/server.js:14:11638)​​
​​[Error]     at WebSocket.<anonymous> (/Users/jyoutsey/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallabye84dcb/server.js:32:8210)​​
​​[Error]     at WebSocket.emit (node:events:526:28)​​
​​[Error]     at Receiver.ontext (/Users/jyoutsey/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallabye84dcb/node_modules/ws/lib/WebSocket.js:841:10)​​
​​[Error]     at /Users/jyoutsey/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallabye84dcb/node_modules/ws/lib/Receiver.js:536:18​​
​​[Error]     at /Users/jyoutsey/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallabye84dcb/node_modules/ws/lib/Receiver.js:368:7​​
​​[Error]     at /Users/jyoutsey/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallabye84dcb/node_modules/ws/lib/PerMessageDeflate.js:249:5​​
​​[Error]     at afterWrite (node:internal/streams/writable:497:5)​​
​​[Error]     at onwrite (node:internal/streams/writable:477:7)​​
​​[Error]     at Zlib.cb (node:internal/streams/transform:202:7)​​
​​[Error]     at Zlib.processCallback (node:zlib:612:8)​​

### Wallaby diagnostics report
Unable to generate?!

**Comment by smcenlly** [comment]
It looks like there are two problems in your issue:

**1. Generating your diagnostics report should not fail**

We believe we have found and fixed the problem, available in Wallaby core `v1.0.1353`.

**2. It looks like perhaps your migration to NX is incomplete and so Wallaby is not finding which tests to run based on your NX configuration.**

We are happy to help you get this working but will definitely require more details of your project, starting with your NX workspace configuration, `package.json`, and a high-level overview of your app structure (libs, apps, etc.).

Can you also please confirm that your tests are running from the CLI with NX? If so, what command are you using to run them? If not, please get them working outside of Wallaby using the NX CLI as a first step.

I also suggest generating a new NX workspace (from scratch) with similar project structure (apps / libs / etc.) and similar testing framework. Confirm that the tests are running with both the command-line and with Wallaby. Once you have done that, you should be able to compare the generated NX configuration to your migration to idenitify why it's not working. If Wallaby is not working with the new dummy workspace you created, please share that with us.

**Comment by jkyoutsey** [comment]
I can run tests manually with 
`npm run test:prod` which results in:
`nx test mms --configuration production --watch false --code-coverage`
And `nx test` works just fine as well.

I forced update to 1.0.1353.
Here is my diagnostics report
```
{
  editorVersion: '1.73.1',
  pluginVersion: '1.0.348',
  editorType: 'VSCode',
  osVersion: 'darwin 22.1.0',
  nodeVersion: 'v16.14.2',
  coreVersion: '1.0.1353',
  checksum: 'OWQxYzlmNTg4MzkxNWFkOGZlMjg5ZmE4NDlmNmFlNjksMTY5Mjc0ODgwMDAwMCww',
  config: { tests: [], files: [] },
  fs: { numberOfFiles: 0 },
  debug: [
    '2022-12-07T23:16:59.998Z config Attempting automatic configuration for angular\n',
    '2022-12-07T23:17:00.020Z angular/cli config Angular CLI not found.\n',
    '2022-12-07T23:17:00.020Z config Finished attempting automatic configuration for angular (22ms)\n',
    '2022-12-07T23:17:00.020Z config Attempting automatic configuration for jest\n',
    "2022-12-07T23:17:00.025Z jest/config Error: Module jest-cli is not found in '<homeDir>/src/worktrees/MyMedstudy.worktrees/TET-2386-f-jky-nx-migration/ng/apps/mms'.\n" +
      '    at Object.<anonymous> (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:31:24913)\n' +
      '    at r (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:31:8061)\n' +
      '    at Object.next (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:31:7356)\n' +
      '    at <homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:31:7265\n' +
      '    at new Promise (<anonymous>)\n' +
      '    at s (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:31:7059)\n' +
      '    at Object.loadJest (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:31:22511)\n' +
      '    at <homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:22845\n' +
      '    at r (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:27578)\n' +
      '    at Object.next (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:26873)\n' +
      '    at <homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:26782\n' +
      '    at new Promise (<anonymous>)\n' +
      '    at s (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:26576)\n' +
      '    at r (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:22533)\n' +
      '    at Object.<anonymous> (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:26253)\n' +
      '    at r (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:27578)\n' +
      '    at Object.next (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:26873)\n' +
      '    at <homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:26782\n' +
      '    at new Promise (<anonymous>)\n' +
      '    at s (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:26576)\n' +
      '    at Object.n [as configure] (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:26134)\n' +
      '    at Config.<anonymous> (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:15:20840)\n' +
      '    at r (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:15:12061)\n' +
      '    at Object.next (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:15:11356)\n' +
      '    at o (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:15:11105)\n',
    '2022-12-07T23:17:00.025Z config Finished attempting automatic configuration for jest (5ms)\n',
    '2022-12-07T23:17:00.025Z config Attempting automatic configuration for vitest\n',
    '2022-12-07T23:17:00.025Z vitest/config Error: Vitest dependency not found.\n' +
      '    at Object.<anonymous> (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:29485)\n' +
      '    at r (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:20:713)\n' +
      '    at Object.next (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:20:8)\n' +
      '    at <homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:31935\n' +
      '    at new Promise (<anonymous>)\n' +
      '    at n (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:31729)\n' +
      '    at Object.r [as configure] (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:28869)\n' +
      '    at Config.<anonymous> (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:15:20840)\n' +
      '    at r (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:15:12061)\n' +
      '    at Object.next (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:15:11356)\n' +
      '    at o (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:15:11105)\n',
    '2022-12-07T23:17:00.026Z config Finished attempting automatic configuration for vitest (0ms)\n'
  ]
}
```

It appears to only try for vitest and jest. But I'm using karma.
My configuration is pretty vanilla. We only have one lib as of now and it's only used for some environment flags.
![image](https://user-images.githubusercontent.com/3846466/206318138-8ad5fc92-0ba3-4ff3-b0cd-6d1197ac0064.png)

package.json
```
{
	"name": "my-app",
	"version": "0.0.0",
	"license": "UNLICENSED",
	"private": true,
	"scripts": {
		"cypress:run": "cypress run --reporter teamcity",
		"cypress:open": "./node_modules/.bin/cypress open",
		"ng": "nx",
		"start": "nx serve mms --hmr",
		"start:local": "nx serve mms -o --proxy-config proxy.conf.local.json",
		"start:local:watch": "nx serve mms -o --proxy-config proxy.conf.local.json --watch",
		"start:remote": "nx serve mms --host JKYMacbookProWork.local --port 4201 --disable-host-check",
		"build:devdeploy": "nx build mms --configuration devdeploy",
		"build:staging": "nx build mms --configuration staging",
		"build:prod": "nx build mms --configuration production",
		"test": "nx test mms",
		"test:prod": "nx test mms --configuration production --watch false --code-coverage",
		"lint": "nx lint",
		"ci:serve": "nx build --configuration production && http-server dist -p 4200",
		"cypress:ci": "start-server-and-test ci:serve http://localhost:4200 cypress:run",
		"postinstall": "ngcc && node ./decorate-angular-cli.js",
		"ci:teamcity-cy-run": "cypress run --reporter mocha-teamcity-reporter"
	},
	"dependencies": {
		"@angular-devkit/core": "^14.2.1",
		"@angular/animations": "~14.2.0",
		"@angular/cdk": "^14.2.0",
		"@angular/common": "~14.2.0",
		"@angular/compiler": "~14.2.0",
		"@angular/core": "~14.2.0",
		"@angular/forms": "~14.2.0",
		"@angular/material": "^14.2.0",
		"@angular/material-moment-adapter": "^14.2.0",
		"@angular/platform-browser": "~14.2.0",
		"@angular/platform-browser-dynamic": "~14.2.0",
		"@angular/router": "~14.2.0",
		"@angular/service-worker": "^14.2.0",
		"@ctrl/tinycolor": "^3.4.0",
		"@ngrx/data": "^14.3.0",
		"@ngrx/effects": "^14.3.0",
		"@ngrx/entity": "^14.3.0",
		"@ngrx/eslint-plugin": "^14.3.2",
		"@ngrx/store": "^14.3.0",
		"@ngrx/store-devtools": "^14.3.0",
		"@nrwl/angular": "^15.3.0",
		"@types/hammerjs": "^2.0.40",
		"blob-util": "^2.0.2",
		"configcat-js": "6.0.0",
		"cypress-downloadfile": "^1.2.3",
		"file-saver": "^2.0.5",
		"hammerjs": "^2.0.8",
		"http-server": "^14.1.0",
		"lodash": "^4.17.21",
		"moment": "^2.29.1",
		"ng-swipe": "^2.0.1",
		"ng2-dragula": "^2.1.1",
		"ngx-color-picker": "^13.0.0",
		"ngx-cookie-service": "^14.0.0",
		"ngx-infinite-scroll": "^14.0.0",
		"reflect-metadata": "^0.1.13",
		"rxjs": "~7.5.0",
		"subsink": "^1.0.2",
		"tslib": "^2.3.0",
		"zone.js": "~0.11.4"
	},
	"devDependencies": {
		"@angular-devkit/build-angular": "~14.2.0",
		"@angular-eslint/eslint-plugin": "14.1.2",
		"@angular-eslint/eslint-plugin-template": "14.1.2",
		"@angular-eslint/template-parser": "14.1.2",
		"@angular/cli": "~14.2.0",
		"@angular/compiler-cli": "~14.2.0",
		"@angular/language-service": "~14.2.0",
		"@ngrx/schematics": "^14.3.0",
		"@nrwl/eslint-plugin-nx": "15.3.0",
		"@nrwl/linter": "15.3.0",
		"@nrwl/nx-cloud": "^15.0.2",
		"@nrwl/workspace": "15.3.0",
		"@types/jasmine": "~4.3.0",
		"@types/jasminewd2": "~2.0.8",
		"@types/lodash": "^4.14.179",
		"@types/node": "^16.0.0",
		"@types/tinycolor2": "^1.4.3",
		"@typescript-eslint/eslint-plugin": "^5.36.1",
		"@typescript-eslint/parser": "^5.36.1",
		"angular2-template-loader": "^0.6.2",
		"cy-verify-downloads": "^0.1.11",
		"cypress": "^10.10.0",
		"cypress-file-upload": "^5.0.8",
		"cypress-log-to-output": "^1.1.2",
		"cypress-teamcity-reporter": "^3.0.0",
		"cypress-xpath": "^1.6.2",
		"eslint": "~8.15.0",
		"eslint-config-prettier": "8.1.0",
		"eslint-plugin-chai-friendly": "^0.7.2",
		"eslint-plugin-cypress": "^2.12.1",
		"eslint-plugin-import": "2.26.0",
		"eslint-plugin-jsdoc": "39.3.2",
		"eslint-plugin-prefer-arrow": "1.2.3",
		"eslint-plugin-prettier": "^4.0.0",
		"jasmine-core": "~4.5.0",
		"jasmine-marbles": "^0.9.0",
		"jasmine-reporters": "^2.5.0",
		"jasmine-spec-reporter": "~7.0.0",
		"karma": "~6.4.0",
		"karma-chrome-launcher": "~3.1.1",
		"karma-coverage": "^2.2.0",
		"karma-jasmine": "~5.1.0",
		"karma-jasmine-html-reporter": "^2.0.0",
		"karma-spec-reporter": "0.0.34",
		"karma-teamcity-reporter": "^1.1.0",
		"ng-mocks": "^14.2.0",
		"ngx-spec": "^2.1.5",
		"node-xlsx": "^0.21.0",
		"nx": "15.3.0",
		"prettier": "^2.6.2",
		"start-server-and-test": "^1.14.0",
		"ts-node": "~7.0.1",
		"typescript": "~4.7.2",
		"wallaby-webpack": "^3.9.16"
	}
}
```

I'm glad to provide any more you need. A comparison of a new project didn't reveal much.

**Comment by smcenlly** [comment]
I'm taking a look at your report now. Quick question: did Wallaby work on the new project?

**Comment by jkyoutsey** [comment]
The new project is apples and oranges. The new version of nx won't generate a non-standalone version of an angular project that I can see. I just don't see HOW to build an Angular workspace with Nx the pre-15.3 way where Angular isn't in standalone mode. The format of the project is different.

**Comment by jkyoutsey** [comment]
No, wallaby does not work when I follow these steps to create a new nx monorepo with an angular project using karma. Wallaby is not TRYING to configure karma AT ALL according to the diagnostics report:

npx create-nx-workspace --preset=empty --name=my-app
cd my-app
npm install @nrwl/angular
nx generate @nrwl/angular:app foobar --style=scss --unitTestRunner=karma

From new project:
```
{
  editorVersion: '1.74.0',
  pluginVersion: '1.0.348',
  editorType: 'VSCode',
  osVersion: 'darwin 22.1.0',
  nodeVersion: 'v16.14.2',
  coreVersion: '1.0.1353',
  checksum: 'OWQxYzlmNTg4MzkxNWFkOGZlMjg5ZmE4NDlmNmFlNjksMTY5Mjc0ODgwMDAwMCww',
  config: { tests: [], files: [] },
  packageJSON: {
    dependencies: {
      '@angular/animations': '~15.0.0',
      '@angular/common': '~15.0.0',
      '@angular/compiler': '~15.0.0',
      '@angular/core': '~15.0.0',
      '@angular/forms': '~15.0.0',
      '@angular/platform-browser': '~15.0.0',
      '@angular/platform-browser-dynamic': '~15.0.0',
      '@angular/router': '~15.0.0',
      '@nrwl/angular': '^15.3.0',
      rxjs: '~7.5.0',
      tslib: '^2.3.0',
      'zone.js': '~0.11.4'
    },
    devDependencies: {
      '@angular-devkit/build-angular': '~15.0.0',
      '@angular-eslint/eslint-plugin': '~15.0.0',
      '@angular-eslint/eslint-plugin-template': '~15.0.0',
      '@angular-eslint/template-parser': '~15.0.0',
      '@angular/cli': '~15.0.0',
      '@angular/compiler-cli': '~15.0.0',
      '@angular/language-service': '~15.0.0',
      '@nrwl/cypress': '15.3.0',
      '@nrwl/eslint-plugin-nx': '15.3.0',
      '@nrwl/linter': '15.3.0',
      '@nrwl/workspace': '15.3.0',
      '@types/jasmine': '~4.0.0',
      '@types/node': '16.11.7',
      '@typescript-eslint/eslint-plugin': '^5.36.1',
      '@typescript-eslint/parser': '^5.36.1',
      cypress: '^11.0.0',
      eslint: '~8.15.0',
      'eslint-config-prettier': '8.1.0',
      'eslint-plugin-cypress': '^2.10.3',
      'jasmine-core': '~4.2.0',
      'jasmine-spec-reporter': '~7.0.0',
      karma: '~6.4.0',
      'karma-chrome-launcher': '~3.1.0',
      'karma-coverage': '~2.2.0',
      'karma-jasmine': '~5.1.0',
      'karma-jasmine-html-reporter': '~2.0.0',
      nx: '15.3.0',
      prettier: '^2.6.2',
      typescript: '~4.8.2'
    }
  },
  fs: { numberOfFiles: 0 },
  debug: [
    '2022-12-08T00:12:24.232Z config Attempting automatic configuration for angular\n',
    '2022-12-08T00:12:24.240Z angular/cli config Angular CLI not found.\n',
    '2022-12-08T00:12:24.240Z config Finished attempting automatic configuration for angular (8ms)\n',
    '2022-12-08T00:12:24.241Z config Attempting automatic configuration for jest\n',
    "2022-12-08T00:12:24.244Z jest/config Error: Module jest-cli is not found in '<homeDir>/src/tmp/nx-test-ng-angular'.\n" +
      '    at Object.<anonymous> (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:31:24913)\n' +
      '    at r (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:31:8061)\n' +
      '    at Object.next (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:31:7356)\n' +
      '    at <homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:31:7265\n' +
      '    at new Promise (<anonymous>)\n' +
      '    at s (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:31:7059)\n' +
      '    at Object.loadJest (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:31:22511)\n' +
      '    at <homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:22845\n' +
      '    at r (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:27578)\n' +
      '    at Object.next (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:26873)\n' +
      '    at <homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:26782\n' +
      '    at new Promise (<anonymous>)\n' +
      '    at s (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:26576)\n' +
      '    at r (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:22533)\n' +
      '    at Object.<anonymous> (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:26253)\n' +
      '    at r (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:27578)\n' +
      '    at Object.next (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:26873)\n' +
      '    at <homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:26782\n' +
      '    at new Promise (<anonymous>)\n' +
      '    at s (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:26576)\n' +
      '    at Object.n [as configure] (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:26134)\n' +
      '    at Config.<anonymous> (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:15:20840)\n' +
      '    at r (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:15:12061)\n' +
      '    at Object.next (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:15:11356)\n' +
      '    at o (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:15:11105)\n',
    '2022-12-08T00:12:24.244Z config Finished attempting automatic configuration for jest (3ms)\n',
    '2022-12-08T00:12:24.244Z config Attempting automatic configuration for vitest\n',
    '2022-12-08T00:12:24.245Z vitest/config Error: Vitest dependency not found.\n' +
      '    at Object.<anonymous> (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:29485)\n' +
      '    at r (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:20:713)\n' +
      '    at Object.next (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:20:8)\n' +
      '    at <homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:31935\n' +
      '    at new Promise (<anonymous>)\n' +
      '    at n (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:31729)\n' +
      '    at Object.r [as configure] (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:19:28869)\n' +
      '    at Config.<anonymous> (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:15:20840)\n' +
      '    at r (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:15:12061)\n' +
      '    at Object.next (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:15:11356)\n' +
      '    at o (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.348/wallaby976a5d/server.js:15:11105)\n',
    '2022-12-08T00:12:24.245Z config Finished attempting automatic configuration for vitest (1ms)\n'
  ]
}
```

**Comment by smcenlly** [comment]
Thank you so much for the instructions, we also hadn't figured out how to change the default from jest.

We're investigating and will get back to you.

**Comment by smcenlly** [comment]
We have found and fixed your problem. We identified that there was a breaking change in one of the later versions of NX workspaces when using Angular.

This has been fixed in Wallaby core `v1.0.1354`. Please update to the latest version, it should fix your issue.
